src/components/Carousel/carousel.module.scss

```diff
@@ -115,7 +115,7 @@
             list-style: none;
             margin: 0;
             overflow: auto;
-            padding: 0;
+            padding: $space-xs;
```

Comment:
do you think the padding should be prop based?
looks like a necessity for cards with drop shadows that get clipped by overflow, but other scenarios won't have that problem
alternatively, could pull that padding into negative margins so that primary content still aligns with the edges as expected